content/en/developers/metrics/_index.md

```diff
@@ -17,20 +18,28 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
+
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric (with [some exception][2]).
+
+A custom metric refers to **a unique combination of metric name, host, and tag values**. In general, any metric you send using [DogStatsD][3], or through a [custom Agent Check][4] is a custom metric. There are no enforced fixed rate limits on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][5].
```

Comment:
I think we're still missing a major section on why custom metrics are useful; - * why do our customers submit custom metrics to us? / what sorts of metrics can be submitted as custom metrics; examples for inspiration: business metrics like revenue or customer order size 

I think we should present the value of custom metrics as part of the overview before diving into how they're unique identified in Datadog

Reply:
We could present the usecase of custom metrics but as a one liner, it's not the purpose of the documentation to address the business perspective of custom metrics, we define them from a technical perspective only.